spec.emu

```diff
@@ -383,6 +383,10 @@ contributors: Google, Ecma International
         1. Let _localeData_ be %DisplayNames%.[[LocaleData]].
         1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, &laquo; `"lookup"`, `"best fit"` &raquo;, `"best fit"`).
         1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Let _calendar_ be ? GetOption(_options_, `"calendar"`, `"string"`, *undefined*, *undefined*).
+        1. If _calendar_ is not *undefined*, then
+          1. If _calendar_ does not match the `(3*8alphanum) *("-" (3*8alphanum))` sequence, throw a *RangeError* exception.
+        1. Set _opt_.[[ca]] to _calendar_.
         1. Let _r_ be ResolveLocale(%DisplayNames%.[[AvailableLocales]], _requestedLocales_, _opt_, %DisplayNames%.[[RelevantExtensionKeys]]).
```

Comment:
It looks like this means the `-u-ca-` in the locale overrides the `{ calendar }` option.  This appears to be consistent with https://github.com/tc39/ecma402/pull/175.

Reply:
no, this only pass the calendar option to the ResolveLocale. Inside ResolveLocale, it will consider that before consider  the "-u-ca-"